### PR TITLE
SLE_MICRO: deploying raw images in VM

### PR DESCRIPTION
**Pull request by tbazant**
### Description

This update describes SLEMCRO raw image deployment in a VMguest
- [x] resolve  profiling on #deployment-ignition-config-file - both for VM and bare-metal deployment

### CODE REVIEW

File: tasks/prepare-alp-vm.xml [hunk 1]
```diff
@@ -20,106 +20,89 @@
     <title>Preparing the &productnameshort; virtual machine</title>
     <abstract>
       <para>
-        This article describes how to configure a new virtual machine suitable
-        for the &productnameshort; deployment.
+        This article describes how to configure a new virtual machine suitable for the
```
Comment (lvicoun) [inline tasks/prepare-alp-vm.xml:23]: Actually, it describes the whole deployment process :)
```suggestion
        This section describes how to prepare a new virtual machine and what steps to take to deploy &productnameshort; on that machine.
```

File: tasks/prepare-alp-vm.xml [hunk 2]
```diff
@@ -20,106 +20,89 @@
...
+        &productnameshort; deployment.
```
Comment (lvicoun) [inline tasks/prepare-alp-vm.xml:24]:
```suggestion
  
```

**Review by lvicoun — CHANGES_REQUESTED** [review]
Thanks for the help, Tomas! Please see my comments. I'll remove the deployment overview later, as it isn't needed.

**Review by dariavladykina — COMMENTED** [review]
Hi, please find some suggestions from me. Thanks!

File: articles/deployment-raw-images-virtual-machines.asm.xml [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+                &productnameshort; provides raw images&mdash;also referred to as <emphasis>pre-built
+                images</emphasis>&mdash;that can be directly deployed to your virtual machine.
+              </para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>WHY?</term>
+            <listitem>
+              <para>
+                Because virtualized deployment saves hardware resources.
```
Comment (tbazant) [inline articles/deployment-raw-images-virtual-machines.asm.xml:161]:
```suggestion
                Virtualized deployment saves hardware resources.
```

**Review by tbazant — COMMENTED** [review]
Suggestions applied

File: tasks/alp-expanding-encrypted-images.xml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      <para>
+        Use the <command>qemu-img</command> command to increase the disk image to the desired size.
+      </para>
+    </step>
+    <step>
+      <para>
+        Set up the virtual machine and boot it. When the &jeosfirstboot; wizard asks you which
+        method to use for encryption, select <guimenu>passphrase</guimenu>.
+      </para>
+    </step>
```
Comment (tbazant) [inline tasks/alp-expanding-encrypted-images.xml:40]:
```suggestion
```

File: tasks/alp-expanding-encrypted-images.xml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    <step>
+      <para>
+        When the system is ready, use the <command>parted</command> command to resize the partition
```
Comment (tbazant) [inline tasks/alp-expanding-encrypted-images.xml:43]:
```suggestion
        Use the <command>parted</command> command to resize the partition
```

**Review by tbazant — COMMENTED** [review]
last 2 fixes

**Pull request merged**